nilearn/decomposition/tests/test_multi_pca.py

```diff
@@ -55,3 +55,46 @@ def test_multi_pca():
 
     # Smoke test to fit with no img
     assert_raises(TypeError, multi_pca.fit)
+
+def test_multi_pca_score():
+    shape = (6, 8, 10, 5)
+    affine = np.eye(4)
+    rng = np.random.RandomState(0)
+
+    # Create a "multi-subject" dataset
+    data = []
+    for i in range(8):
+        this_data = rng.normal(size=shape)
+        # Create fake activation to get non empty mask
+        this_data[2:4, 2:4, 2:4, :] += 10
+        data.append(nibabel.Nifti1Image(this_data, affine))
+
+    mask_img = nibabel.Nifti1Image(np.ones(shape[:3], dtype=np.int8), affine)
+
+    # Assert that score is between zero and one
+    multi_pca = MultiPCA(mask=mask_img, random_state=0, memory_level=0, n_components=3)
+    multi_pca.fit(data)
+    s = multi_pca.score(data, per_component=False)
+    np.testing.assert_array_less(s, np.ones(6))
+    np.testing.assert_array_less(-s, np.ones(6))
+
+    # Assert that score does not fail with single subject data
+    multi_pca = MultiPCA(mask=mask_img, random_state=0, memory_level=0, n_components=3)
+    multi_pca.fit(data[0])
+    s = multi_pca.score(data, per_component=False)
+    assert_is_instance(s, float)
+    assert(0. <= s <= 1.)
+
+    # Single subject data
+    single_data = rng.normal(size=shape)
+    # Create fake activation to get non empty mask
```

Comment:
This is just to mask sure that our data is not masked (data masking heuristic is hard coded)